### PR TITLE
Patch emulators on NixOS for upgrade tests

### PR DESCRIPTION
**Pull request by tsusanka**
Let me know if this is the nix way.

### CODE REVIEW

File: tests/download_emulators.sh [hunk 1]
```diff
@@ -3,6 +3,10 @@ SITE="https://firmware.corp.sldev.cz/upgrade_tests/"
 cd "$(dirname "$0")"
 
 # download all emulators without index files, without directories and only if not present
-wget --no-verbose --no-clobber --no-parent --cut-dirs=2 --no-host-directories --recursive --reject "index.html*" -P emulators/ $SITE
+wget -e robots=off --no-verbose --no-clobber --no-parent --cut-dirs=2 --no-host-directories --recursive --reject "index.html*" -P emulators/ $SITE
 
 chmod u+x emulators/trezor-emu-*
+
+if [[ `uname -a` =~ "NixOS" ]]; then
```
Comment (prusnak) [inline tests/download_emulators.sh]: Maybe just replace with `[ -f /etc/NIXOS ];`

File: tests/download_emulators.sh [hunk 2]
```diff
@@ -3,6 +3,10 @@ SITE="https://firmware.corp.sldev.cz/upgrade_tests/"
...
+  cd emulators && nix-shell --run "autoPatchelf trezor-emu* > /dev/null 2>&1"
```
Comment (prusnak) [inline tests/download_emulators.sh]: Why not just using plain
```
nix-shell --run "autoPatchelf trezor-emu*"
```
?

Is there any benefit to redirecting the outputs?

File: tests/download_emulators.sh (hunk 1 above)
Comment (tsusanka) [reply tests/download_emulators.sh]: Fixed, surprisingly it failed on CI anyway.

File: tests/download_emulators.sh (hunk 2 above)
Comment (tsusanka) [reply tests/download_emulators.sh]: I've copied it from adminkos to be honest. It is true that the output is quite verbose, but I guess they hid it for "ordinary" users and we actually want the output. Fixed.

**Comment by prusnak** [comment]
LGTM what do you think @mmahut?

**Comment by prusnak** [comment]
Merged

**Pull request merged**